src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -36,6 +36,11 @@ public static List<HearingChannel> getHearingChannels(@Valid SscsCaseData caseDa
     }
 
     public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
+
+        if(caseData.getAdjournCaseTypeOfNextHearing() != null) {
+            return caseData.getAdjournCaseTypeOfNextHearing();
```

Comment:
Could you please add some logging here? This also needs a test

Reply:
I've added a test case but what needs to be logged? @dattipoeHMCTS